src/lib/Behat/PageElement/UniversalDiscoveryWidget.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Behat\PageElement;
+
+use EzSystems\EzPlatformAdminUi\Behat\Helper\UtilityContext;
+use PHPUnit\Framework\Assert;
+
+class UniversalDiscoveryWidget extends Element
+{
+    public function __construct(UtilityContext $context)
+    {
+        parent::__construct($context);
+        $this->fields = [
+            'tabSelector' => '.c-tab-nav-item',
+            'confirmButton' => '.m-ud__action--confirm',
+            'selectContentButton' => '.c-meta-preview__btn--select',
+            'elementSelector' => '.c-finder-tree-branch:nth-of-type(%d) .c-finder-tree-leaf',
+        ];
```

Comment:
Actually it's a good point - we should define fields in this way rather than through defining new `$fields` variable. Please change it also in other places.